src/streams/streamsActions.js

```diff
@@ -43,6 +43,14 @@ export const updateExistingStream = (
   if (initialValues.invite_only !== newValues.isPrivate) {
     await updateStream(getAuth(getState()), id, 'is_private', newValues.isPrivate);
   }
+  if (initialValues.is_announcement_only !== newValues.isAnnouncementOnly) {
```

Comment:
This line of code looks OK but I am starting to be concerned we are calling `updateStream` too many times. Is there a 'batch update' version of this API call?

Reply:
submitted #3224 :)